scripts/dev/templates/agent/Dockerfile.ubi

```diff
@@ -1,10 +1,12 @@
 {% extends "Dockerfile.template" %}
 
-{% set base_image = "registry.access.redhat.com/ubi7/ubi" %}
+{% set base_image = "registry.access.redhat.com/ubi8/ubi-minimal" %}
 
 {% block packages -%}
-RUN yum install -y  --disableplugin=subscription-manager -q curl \
-    hostname nss_wrapper --exclude perl-IO-Socket-SSL procps \
-    && yum upgrade -y -q \
+RUN microdnf install -y  --disableplugin=subscription-manager curl \
+    hostname nss_wrapper tar gzip procps\
+    && microdnf upgrade -y  \
     && rm -rf /var/lib/apt/lists/*
+
+RUN microdnf remove perl-IO-Socket-SSL
```

Comment:
We do the same in other places: `microdnf` doesn't have an `exclude` option, so we install and then remove